### ISSUE TITLE
GetGraphicsAdapter only finds iGPU

### ISSUE THREAD

**Issue by IronHunter1** [dynamiquel/NVIDIA-Ansel-AI-Enhancer#10]
My laptop (Razer Blade Pro) kept showing an error message because only the iGPU was recognized. The List of GPUs before returning graphicsAdapter[0] only contained the iGPU aswell.
I tried to fix it and ended up with this. Seems to work on my System, but ive got no other computer to test it on.

[fix.txt](https://github.com/dynamiquel/NVIDIA-Ansel-AI-Enhancer/files/7815630/fix.txt)

**Comment by dynamiquel** [comment]
So I notice two things.
First, `graphicsAdapters.OrderBy(o => o.SupportLevel).Reverse().ToList();` becoming `graphicsAdapters = graphicsAdapters.OrderBy(o => o.SupportLevel).Reverse().ToList();`. I did notice this before and could have sworn I changed it since I felt like an idiot.
The second is using 'Win32_VideoController'. 

Before this gets implemented, can you confirm that the second change is required? Is the problem fixed from just the first change?

**Comment by IronHunter1** [comment]
For me it wasnt fixed from just the first one. Is there anything wrong with using Win32_VideoController? I dont usually work with WQL

**Comment by teherauto** [comment]
Same problem with RTX3070 and Intel UHD on a Schenker Key 17 (E21) laptop. 
Set the Nvidia adapter on Nvidia Control panel, on the Windows 10 Graphics settings page, but the app always start with Intel UHD.

**Comment by Stridsvagn69420** [comment]
Similar issue here. Despite telling Windows to use the dGPU, the program seems to start with my iGPU. The good news is that the Nvidia GPU Activity thing tells me that it actually runs on my 3070 and then upscaling something (with force mode enabled) makes my GPU do something and Task-Manager tells me it's also using the 3070.

TL;DR: It does work, but I have to use Force Mode, because it still thinks it runs on my iGPU.
